src/server/api/common_api.py

```diff
@@ -7,11 +7,23 @@
 @common_api.route('/api/contacts/<search_text>', methods=['GET'])
 def get_contacts(search_text):
     with engine.connect() as connection:
-        #todo: add logic to grab names from all sources - 1.salesforce 2.volgistics 3.petpoint once you find one, return all unique names and email
-        query = text("select concat(first_name,' ',last_name) as name, email, contact_id from salesforcecontacts \
-            WHERE lower(first_name) like :search_text \
-            OR lower(last_name) like :search_text")
-        query_result = connection.execute(query, search_text='%{}%'.format(search_text.lower()))
+        search_text = search_text.lower()
+
+        names = search_text.split(" ")
```

Comment:
@dkelley We are currently working on a new table - user_info that will have the full name in one field, so this won't be necessary.  Would be best if you sync with @JoeIlluminati on this one and see when we are going to start using user_info to not add unnecessary complexities.

Reply:
Thanks @urirot, @JoeIlluminati happy to chat, and you may want to consider having first and last name in the user info table to allow for improved searching like above.  When this data grows, a wildcard match may not be enough to find an entry.  The above update was a request from @ckohl226, and was really just a way for me to learn the system and get my hands dirty so I'm not upset if this was throw away and gets ditched if you go to user_info.

Reply:
@dkelley 
Sure, not a bad thing to have an improved search for now until the release and updates of the user_info table. Thanks :)
Also first + last name might make sense for the user_info table. can you also take this one with @JoeIlluminati ?

Reply:
User could add names in any order so a slight modification 

`WHERE (lower(name_1) like :first_name AND lower(name_2) like :last_name ) OR (lower(name_1) like :last_name AND lower(name_2) like :first_name) order by last_name`

Reply:
I tried this and didn't work. @sposerina you can try adding this. also maybe we want to wait for user_info before improving this search further. 
for now I think first name and then last name for full name search is good enough

Reply:
Yup. I agree.